### PR TITLE
corrected the formula under how the committer could cheat 

### PR DESCRIPTION
**Pull request by tanim0la**

### CODE REVIEW

File: content/pedersen-commitment/en/pedersen-commitment.md [hunk 1]
```diff
@@ -79,20 +79,41 @@ $$
 to a different $(v', s')$ other than the value they originally committed.
 
 Here's how the committer could cheat if they know that $g$ is the discrete logarithm of $G$ and $b$ is the discrete logarithm of $B$.
+$$
+\text{B} = bG
```
Comment (jeffreyscholz) [inline content/pedersen-commitment/en/pedersen-commitment.md]: can be just B with no text. It's a math variable, so not text. same as below

File: content/pedersen-commitment/en/pedersen-commitment.md [hunk 2]
```diff
@@ -79,20 +79,41 @@ $$
 to a different $(v', s')$ other than the value they originally committed.
 
 Here's how the committer could cheat if they know that $g$ is the discrete logarithm of $G$ and $b$ is the discrete logarithm of $B$.
+$$
+\text{B} = bG 
+$$
+$$
+\text{G} = gB
```
Comment (jeffreyscholz) [inline content/pedersen-commitment/en/pedersen-commitment.md]: same

**Review by jeffreyscholz — CHANGES_REQUESTED** [review]
no need to have \text{} for math objects

**Comment by tanim0la** [comment]
> no need to have \text{} for math objects

done

**Review by jeffreyscholz — APPROVED** [review]

**Pull request merged**